<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbOriginalImageDisplay">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>132</y>
      <width>284</width>
      <height>284</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_c">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>132</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_m">
    <property name="geometry">
     <rect>
      <x>474</x>
      <y>132</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_y">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>318</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_k">
    <property name="geometry">
     <rect>
      <x>474</x>
      <y>318</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbColor1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbColor2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbColor3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbColor4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBstartLayering">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBloadImage">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>132</y>
      <width>284</width>
      <height>284</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dSBcAdjustmentCoefficient">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dSBmAdjustmentCoefficient">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dSByAdjustmentCoefficient">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dSBkAdjustmentCoefficient">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBsaveScheme">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>18</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>44</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>18</y>
      <width>28</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>方案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNewScheme">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>18</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>44</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbStartInScheme">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>24</y>
      <width>107</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>启动时打开方案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOpenScheme">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>18</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>44</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBsaveSchemeAs">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>18</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>保存为</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSchemeDameDisplay">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBexportVectorDiagram">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>导出矢量图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBexportGrayScaleImage">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>导出灰度图</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_r">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>132</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_g">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>132</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbOriginalImageDisplay_b">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>322</y>
      <width>138</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>通用设置</string>
    </property>
    <widget class="QSpinBox" name="sBimageHeight">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图片高度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>像素网格高度：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSBpixelGridheight">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>2.000000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="cBblackLayering">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>黑色分层次</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbColorLayerdType">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CMYK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RGB</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>分层类型：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>导出灰度图设置</string>
    </property>
    <widget class="QRadioButton" name="rBrandomSorting">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>随机排序</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBsortNeatly">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>整齐排序</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBaggregateSort">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>聚集排序</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cBSelectC">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBSelectM">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBSelectY">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBSelectK">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBSelectG">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBSelectB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBSelectR">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="hSColorSaturation">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>421</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>饱和度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbColorSaturationDisplay">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cbGridType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>圆点</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>方点</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>472</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>网点：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbImageName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBdisplayImage1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBdisplayImage2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBdisplayImage3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBdisplayImage4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
